packages/plugin/src/our-options.ts

```diff
@@ -328,6 +331,9 @@ export function makeInternalOptions(
     if (params?.add_pb_suffix) {
         o.addPbSuffix = true;
     }
+    if (params?.force_disable_services) {
+      o.forceDisableServices = true;
+    }
```

Comment:
I wonder if we should set `forcedServerStyle` and `forcedClientStyle` here to be explicit, but can't think of a case the very obvious if-statement below wouldn't catch.